src/core/ext/filters/client_channel/lb_policy/pick_first/pick_first.cc

```diff
@@ -391,42 +416,8 @@ void PickFirst::UpdateLocked(const grpc_channel_args& args,
           ->CheckConnectivityStateAndStartWatchingLocked();
```

Comment:
Now that we are checking the subchannels' state above, we can just call `StartConnectivityWatchLocked()` here.

Reply:
Good catch.  Done.